### PR TITLE
change status to test

### PR DESCRIPTION
**Pull request by frack113**
- Change status `experimental` to `test` for old rules
- Change '2' spaces to '4' 
- Split description when it is a no end string (more easy to read)

### CODE REVIEW

**Review by phantinuss — APPROVED** [review]

**Comment by nasbench** [comment]
Don't merge. Until my PR is merged. I renamed some files and I don't one want conflicts :D

**Pull request merged**
